code/game/gamemodes/antag_spawner.dm

```diff
@@ -70,3 +70,83 @@
 	ticker.mode.update_synd_icons_added(R.mind)
 	R.mind.special_role = "syndicate"
 	R.faction = list("syndicate")
+
+/obj/item/weapon/antag_spawner/slaughter_demon //Warning edgiest item in the game
+	name = "vial of blood"
+	desc = "A magically infused bottle of blood, distilled from countless murder victims. Used in unholy rituals to attract horrifying creatures."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "vial"
+	var/checking = 0
+
+
+/obj/item/weapon/antag_spawner/slaughter_demon/attack_self(mob/user as mob)
```

Comment:
not really sure this should be handled this way--this is a wizard item, so it may be best to handle it just like the revenant does; pull someone and shove them into the slaughter demon; also means you dont' ahve to have the get_candidate_answer stuff.